Dockerfile

```diff
@@ -75,6 +75,10 @@ COPY --from=build /c3po/pcrf/tssf/bin /bin
 
 ## Stage hssdb
 FROM cassandra:2.1.20 as hssdb
```

Comment:
Seems like changing to 2.1 works without the workaround. This way we get critical fixes too?
```suggestion
FROM cassandra:2.1 as hssdb
```

Reply:
Sure. I'll close this PR and open another one

Reply:
Actually I'll just commit your suggestions